asv.conf.json

```diff
@@ -53,9 +53,11 @@
     // followed by the pip installed packages).
     //
     "matrix": {
+//        "numpy": ["1.17", "1.18", "1.19", ""],
```

Comment:
I think testing against more numpy versions (latest we support vs. oldest we support) might be more interesting than python versions; but as it stands, the test suite takes long enough to run that probably don't need to worry about adding more test configurations.

Reply:
Yeah, agreed. If I can pare down the combinations, numpy would be the next variable I'd want to add.